ivy/functional/ivy/creation.py

```diff
@@ -382,22 +408,31 @@ def eye(
     n_rows
         number of rows in the output array.
     n_cols
-        number of columns in the output array. If None, the default number of columns in
-        the output array is equal to n_rows. Default: None.
+        number of columns in the output array.
+        If None, the default number of columns in the output array is
+        equal to n_rows.Default: None.
     k
-        index of the diagonal. A positive value refers to an upper diagonal, a negative
-        value to a lower diagonal, and 0 to the main diagonal. Default: 0.
+        index of the diagonal.
+        A positive value refers to an upper diagonal,
+        a negative value to a lower diagonal,
+        and 0 to the main diagonal. Default: 0.
     dtype
-        output array data type. If dtype is None, the output array data type must be the
-        default floating-point data type. Default: None.
+        output array data type.
+        If dtype is None, the output array data type must be the default floating-
+        point data type. Default: None.
     device
          device on which to place the created array.
```

Comment:
Please update the docstring according to [this](https://lets-unify.ai/ivy/contributing/12_docstrings.html)